libyui-rest-api/src/YHttpWidgetsHandler.cc

```diff
@@ -57,6 +66,4 @@ void YHttpWidgetsHandler::process_request(struct MHD_Connection* connection,
         body << "{ \"error\" : \"No dialog is open\" }" << std::endl;
         error_code = MHD_HTTP_NOT_FOUND;
     }
-
-    content_type = "application/json";
```

Comment:
Um, it seems that the content type is not set for the error message at line 57 / 66 above this. Or did I overlook something?

Reply:
good catch, fixed also in the action handler.